kroxylicious/src/main/resources/log4j2.properties

```diff
@@ -11,11 +11,23 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} <%t> %-5p %c:%L - %m%n
 
-rootLogger.level = WARN
+rootLogger.level = ${env:KROXYLICIOUS_ROOT_LOG_LEVEL:-WARN}
 rootLogger.appenderRefs = stdout
 rootLogger.appenderRef.console.ref = STDOUT
 rootLogger.additivity = false
 
-#logger.netty.name = io.netty
-#logger.netty.level = TRACE
-#logger.netty.additivity = false
+# The DownstreamNetworkLogger and DownstreamFrameLogger loggers are required to allow
+# the virtual cluster logNetwork/logFrames feature to work.  The log level needs to be in sync
+# with the logger level used in the code.
+logger.DownstreamNetworkLogger.name = io.kroxylicious.proxy.internal.DownstreamNetworkLogger
```

Comment:
Does the existence of `Downstream*Logger` imply that we should also have a `Upstream*Logger` ?

Reply:
Yeah, so checking why `io.kroxylicious.proxy.internal.DownstreamNetworkLogger` doesn't appear in the diff outside of the log file confirms that `io.kroxylicious.proxy.internal.UpstreamNetworkLogger` is a thing so can we add that here as well?